adafruit_mpl3115a2.py

```diff
@@ -130,33 +132,47 @@ class MPL3115A2:
     # creates a flag in _MPL3115A2_REGISTER_STATUS that we were not clearing depending
     # on the properties reading order
 
+    _device_id = UnaryStruct(_MPL3115A2_WHOAMI, "B")
+    _device_control = UnaryStruct(_MPL3115A2_CTRL_REG1, "B")
+    _data_config = UnaryStruct(_MPL3115A2_PT_DATA_CFG, "B")
+    _register_status = UnaryStruct(_MPL3115A2_REGISTER_STATUS, "B")
+    _sea_level_pressure_msb = UnaryStruct(_MPL3115A2_BAR_IN_MSB, "B")
+    _sea_level_pressure_lsb = UnaryStruct(_MPL3115A2_BAR_IN_LSB, "B")
+
+    _reset_status = RWBits(1, _MPL3115A2_CTRL_REG1, 2)
```

Comment:
Any reason why you didn't use `adafruit_register.i2c_bit.RWBit`?  It looks like it should be good here since these are descriptors for one bit.

Reply:
Yes, you are right, no reason other than stron habits die hard :). I will change it